FIELDS.md

```diff
@@ -6,16 +6,14 @@
 ### General output information:
 <p style="margin-left: 40px"> Missing data in the annotation is marked by <strong>'!'</strong>  </p>
 <p style="margin-left: 40px"> Multiple values for a single annotated position are separated by <strong>';'</strong> </p>
+<p style="margin-left: 40px"> Multiple values for a single annotated position that are nested within <strong>';'</strong> delimited values (e.g 2D values) are separated by <strong>ASCII UNIT SEPARATOR (ASCII code 31)</strong> </p>
```

Comment:
it might help to give a small example of when such a structure might occur, and what the data would look like in that case

Reply:
Good suggestion, will take care of that in the morning.